libs/vkd3d/device.c

```diff
@@ -465,11 +465,13 @@ static const struct vkd3d_instance_application_meta application_override[] = {
      * This works okay with zerovram on first game boot, but not later, since this memory is guaranteed to be recycled.
      * Game also relies on indirectly modifying CBV root descriptors, which means we are forced to rely on RAW_VA_CBV.
      * It also relies on multi-dispatch indirect with state updates which is ... ye.
-     * Need another config flag to workaround that as well. */
+     * Need another config flag to workaround that as well.
+     * Poor loading times and performance with ReBar on some devices.
```

Comment:
@Etaash-mathamsetty Heya, just poking as I noticed this was fixed in-game and went looking for what fixed it. Thanks for the addressing :heart: 

I'm curious how you narrowed down long load times in Halo Infinite to this fix? Any good tips to debug? Is there discussion/context around this problem somewhere?

Reply:
it was found by accident

Reply:
It had a similar effect in AoW4 and was tried and tested a bunch for this game too when the issue was being looked in to.